docs/guides/cli/20_proxy-and-tunnel.mdx

```diff
@@ -323,13 +321,17 @@ ENTRYPOINT ["/docker-entrypoint.sh"]
 
 ## Ory Tunnel
 
-The Ory Tunnel has the same functionality and behaves the same way 
-as the Proxy with these exceptions:  
+The main role of the Ory Tunnel is exposing the Ory APIs under the same top-level domain 
+as your application to ensure that there are no CORS issues.
+
+The Tunnel shares most of its code with the Ory Proxy, and from the user's point of view
+it works in a similar way. There are, however, several important differences between these tools: 
 
+  ❌ Ory Tunnel doesn't handle traffic routing
```

Comment:
what does that mean?

Reply:
The tunnel doesn't act as an entry point to the app the way I understand it so you'd call the app directly instead of the proxy. Then the app gets to the Ory endpoints through the tunnel. With the proxy, the traffic hits the proxy and it then "manages" the request, takes it to the app and then to the endpoints. That's what I got after talking to you and PM. Does that make sense?

Reply:
From `--help`:

```
It is similar to the `ory proxy` command, but it does not require to
route all your application's traffic through it!
```

Reply:
i see